### PR TITLE
Minor grammar fixes.

### PR DESCRIPTION
**Pull request by Ujjwaldeep**
Just fixed grammar in ReadMe.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,17 +25,17 @@ For details, see the User's Manuals:
...
 
 Bond C++ library requires some C++11 features (currently limited to those
-supported bv Visual C++ 2013); a C++11 compiler is required. Additionally,
+supported by Visual C++ 2013); a C++11 compiler is required. Additionally,
 to build Bond you will need CMake (3.1+),
 [Haskell Stack](https://docs.haskellstack.org/en/stable/README/#how-to-install)
 (1.5.1+) and Boost (1.61+).
 
-Additionally, Bond requires RapidJSON and optionally requires gRPC. The Bond repository primarily uses Git submodules for these two dependencies. It should be cloned with the `--recursive` flag:
+Additionally, Bond requires RapidJSON and optionally requires gRPC. The Bond repository primarily uses Git submodules for these two dependencies. It should be closed with the `--recursive` flag:
```
Comment (chwarr) [inline README.md]: "cloned" was correct here: it's the Git command that needs to be used. Can you revert this change?

File: README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ sudo apt-get install \
 Additionally, you need the [Haskell Tool
 Stack](https://docs.haskellstack.org/en/stable/README/). If your distro isn't
 shipping a new enough version of it, you may encounter some non-obvious build
-failures, so we recommend installing the latest Stack outside of package
+failures, so we recommend installing the latest Stack outside of the package
```
Comment (chwarr) [inline README.md]: To my ear, this reads better with out the additional "the". Can you please revert its addition?

Comment (Ujjwaldeep) [reply README.md]: sure, let me do that. But I need to get this merged for hacktoberfest. This pull will be counted in the challenge.

**Comment by Ujjwaldeep** [comment]
Thank you for merging the changes. I Appreciate it.
